### PR TITLE
chore: update trivy version Dockerfile

### PR DESCRIPTION
**Pull request by oranmoshai**

### CODE REVIEW

**Review by knqyf263 — CHANGES_REQUESTED** [review]
Can you update README.md to align with the new option?
https://github.com/aquasecurity/trivy/discussions/1571

Before:

```
      - name: Run Trivy vulnerability scanner
        uses: aquasecurity/trivy-action@master
        with:
          image-ref: 'docker.io/my-organization/my-app:${{ github.sha }}'
          format: 'template'
          template: '@/contrib/sarif.tpl'
          output: 'trivy-results.sarif'
```

After:

```
      - name: Run Trivy vulnerability scanner
        uses: aquasecurity/trivy-action@master
        with:
          image-ref: 'docker.io/my-organization/my-app:${{ github.sha }}'
          format: 'sarif'
          output: 'trivy-results.sarif'
```

https://github.com/aquasecurity/trivy-action#using-trivy-with-github-code-scanning
https://github.com/aquasecurity/trivy-action#using-trivy-to-scan-your-git-repo
https://github.com/aquasecurity/trivy-action#using-trivy-to-scan-infrastucture-as-code

and here
https://github.com/aquasecurity/trivy-action#inputs

**Comment by knqyf263** [comment]
Oh, looks like we also have to fix here.
https://github.com/aquasecurity/trivy-action/blob/9c21d3ca2c14eb35419e2a8b66d1195946d579b8/entrypoint.sh#L141-L144

**Review by tzurielweisberg — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -389,8 +381,7 @@ Following inputs can be used as `step.with` keys:
 | `input`          | String  |                                    | Tar reference, e.g. `alpine-latest.tar` |
 | `image-ref`      | String  |                                    | Image reference, e.g. `alpine:3.10.2`         |
 | `scan-ref`       | String  | `/github/workspace/`               | Scan reference, e.g. `/github/workspace/` or `.`|
-| `format`         | String  | `table`                            | Output format (`table`, `json`, `template`)   |
-| `template`       | String  |                                    | Output template (`@/contrib/sarif.tpl`, `@/contrib/gitlab.tpl`, `@/contrib/junit.tpl`)|
```
Comment (knqyf263) [inline README.md:393]: `template` is still valid. We just removed `sarif.tpl`.

File: entrypoint.sh [hunk 2]
```diff
@@ -140,7 +140,7 @@ returnCode=$?
 # This is a feature, not a bug :)
 if [[ ${template} == *"sarif"* ]]; then
```
Comment (knqyf263) [inline entrypoint.sh:141]: We have to fix this line as well.

**Review by knqyf263 — CHANGES_REQUESTED** [review]

**Comment by simar7** [comment]
> Anything else? @simar7 @krol3

lgtm, module the feedback you left!

**Review by simar7 — CHANGES_REQUESTED** [review]
overall lgtm, just the comments @knqyf263 mentioned.

**Comment by knqyf263** [comment]
Oran is under weather and I've updated instead. v0.23.0 contains some breaking changes. I want to make sure this update works properly. Could you test this branch? @simar7 @krol3

**Comment by krol3** [comment]
@knqyf263 , @simar7, I updated in my branch that has unit test using this latest version (0.23), It seems to work the parameters using sarif. Any feedback to improve the unit test. please take a look into in https://github.com/aquasecurity/trivy-action/pull/85

**Review by knqyf263 — APPROVED** [review]

**Review by knqyf263 — APPROVED** [review]

**Review by simar7 — APPROVED** [review]

**Pull request merged**